quickstack/manifests/controller_common.pp

```diff
@@ -881,4 +896,14 @@
     priv_net     => $priv_net,
     #before => Class['hosts'],
   }
+
+
+  if hiera('moc::clusterdeployment') == 'true' {
+    class {'::galera::server':
+      mysql_root_password => $mysql_root_password,
+      require => Class["moc_openstack::ha"],
+    }
+    class {'moc_openstack::ha':
+    }
+  }
 }
```

Comment:
We don't need quickstack/manifests/galera.unused/ directory. Do we want to keep it for future reference?